haystack/components/evaluators/sas_evaluator.py

```diff
@@ -11,7 +11,7 @@
 from haystack.utils import ComponentDevice, expit
 from haystack.utils.auth import Secret, deserialize_secrets_inplace
 
-with LazyImport(message="Run 'pip install scikit-learn \"sentence-transformers>=2.2.0\"'") as sas_import:
+with LazyImport(message="Run 'pip install \"sentence-transformers>=2.3.0\"'") as sas_import:
```

Comment:
unrelated: we don't directly depend on `scikit-learn` (which is a core dependency of `sentence-transformers`).